straxen/plugins/nveto_recorder.py

```diff
@@ -16,8 +16,8 @@
                  help="Pretrigger time before coincidence window in ns."),
     strax.Option('resolving_time_recorder_nv', type=int, default=600,
                  help="Resolving time of the coincidence in ns."),
-    strax.Option('nbaseline_samples_lone_records_nv', type=int,
-                 default_by_run=[(0, 10), (12684, 26)], track=True,
+    strax.Option('nbaseline_samples_lone_records_nv', type=tuple,
```

Comment:
@WenzDaniel should we not change this option to be the same as in the veto_pulse_processing.py: 
```baseline_samples_nv``` 

Now there are two names for the same thing

Reply:
Yes, true I forgot to mention this. Would be nice if you could also this into baseline_samples_nv Aaron. Thanks a lot.

Reply:
done